Intersect.Server/Entities/Events/CommandProcessing.cs

```diff
@@ -707,10 +707,21 @@ Stack<CommandInstance> callStack
             Stack<CommandInstance> callStack
         )
         {
-            player.Warp(
-                command.MapId, command.X, command.Y,
-                command.Direction == WarpDirection.Retain ? (byte) player.Dir : (byte) (command.Direction - 1)
-            );
+            if (command.ChangeInstance)
+            {
+                player.Warp(
+                    command.MapId, command.X, command.Y,
+                    command.Direction == WarpDirection.Retain ? (byte)player.Dir : (byte)(command.Direction - 1),
+                    false, 0, false, false, command.InstanceType
```

Comment:
Do not add `false, 0, false, false` just put `mapInstanceType: command.InstanceType`.

Honestly the entire method could be changed to just be 
```csharp
player.Warp(
    command.MapId, command.X, command.Y,
    command.Direction == WarpDirection.Retain ? (byte)player.Dir : (byte)(command.Direction - 1),
    mapInstanceType: command.ChangeInstance ? command.InstanceType : null
);
```

With `MapInstanceType mapInstanceType` being changed to `MapInstanceType? mapInstanceType`

Reply:
Cool, done - with the exception that having a ternary in the function params is not supported in C# 7.3